tests/test_visitors/test_ast/test_attributes/test_magic_attributes.py

```diff
@@ -107,6 +108,24 @@ def method(cls):
         super().{0}()
 """
 
+magic_wrapper_method = """
```

Comment:
Should we allow cases like:

```python
def __init__(...):
    other.__init__(...)
```

?

I think it makes sense for `__getattr__` and `__dir__` which can be used on a module level. But, not for anything else. Do you agree?

Reply:
Do you want to allow only a limited subset of _magic_ methods here? With an explicitly provided list?

I do not see how that can significantly improve the utility of this feature.